README.rst

```diff
@@ -0,0 +1,325 @@
+============================================
+Batch Data Processing in CDAP using Workflow
```

Comment:
Perhaps a better title, because you probably want to high-lite the use of a predicate?

Predicate Workflow Processing in CDAP ?